recipes/biotransformer/build.sh

```diff
@@ -5,7 +5,10 @@ outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
 cp -R * $outdir/
-cp $RECIPE_DIR/biotransformer.py $outdir/biotransformer
+cp $RECIPE_DIR/biotransformer.py $outdir/
 ls -l $outdir
+printf '#!/bin/bash\n' > $outdir/biotransformer
+printf "cd ${outdir} \n" >> $outdir/biotransformer
+printf 'python biotransformer.py "$@"\n' >> $outdir/biotransformer
```

Comment:
I was thinking that the `cd` could break passing relative paths to `biotransformer`, would that work instead?

```suggestion
printf 'python ${outdir}/biotransformer.py "$@"\n' >> $outdir/biotransformer
```

Reply:
@nsoranzo in my limited testing `${outdir}` will not expand when in single quotes

Reply:
I haven't added single quotes, sorry I don't understand.

Reply:
@nsoranzo  but you added the `${outdir}` expansion and we are within the single quotes. Or am I missing something?

Reply:
Aside from the single quotes, in my testing the python script called from bash with full path will still operate on cwd (not the directory where the .py script is).

iow the following code with fail unable to find the `/Users/marten/miniconda3/share/biotransformer-1.1.5-0/supportfiles/` folder.

```
#!/usr/bin/env bash
python /Users/marten/miniconda3/share/biotransformer-1.1.5-0/biotransformer "$@"
```